content/blog/2019/07/2019-07-02-plugin-management-tool-alpha-release.adoc

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Plugin Management Library and CLI Tool Alpha Release"
+tags:
+- plugins
+- pluginmanagement
```

Comment:
```suggestion
- plugin-management
```

Reply:
Are you sure there should be a dash here? 
https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-blog-post